Src/Nodes/abstract_node.py

```
@@ -39,7 +39,7 @@ class AbstractNode(ABC):
 
 
     def __init__(self, node_tag: int | str, annotations: dict[str: type], \
-                 logic: Callable, docs: str = None):
+                 logic: Callable, docs: str = None, resize_callback: Callable = None):
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)